src/content/docs/zh-cn/guides/cms/spinal.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Spinal & Astro
+description: 将 Prismic 作为 CMS 添加到您的 Astro 项目中。
+type: cms
+stub: true
+service: Spinal
+i18nReady: true
+---
+
+[Spinal](https://spinalcms.com/cms-for-astro/) 是一款商业化的、专注于 SaaS、基于 Git 的 CMS。
+
+## 快速入门
+
+1. [创建一个 Spinal 账户](https://spinalcms.com/signup/)。
+2. 连接你的 GitHub 和 Spinal 账户。
+3. 选择你的 Astro 项目存储库。
+
+在完成上述步骤后，在你所选文件夹中所有的 Markdown 内容都会被导入到你的 Spinal 账户，你可以 Spinal 编辑器中直接编辑。
+
+## 官方资源
+
+- [Documentation theme built for Astro with Tailwind CSS](https://spinalcms.com/resources/astro-documentation-theme-with-tailwind-css/)
```

Comment:
```suggestion
- [使用 Tailwind CSS 为 Astro 构建文档主题](https://spinalcms.com/resources/astro-documentation-theme-with-tailwind-css/)
```

Reply:
Thank you very much for patience in correcting personal pronouns!